debian/changelog

```diff
@@ -1,10 +1,11 @@
-cloud-init (22.3-3-g9f0efc47-0ubuntu1~22.04.1) UNRELEASED; urgency=medium
+cloud-init (22.3-3-g9f0efc47-0ubuntu1~22.04.1) jammy; urgency=medium
```

Comment:
Not sure why this specifcally is showing up in the visual github visual diff. I don't see it on the cmdline against upstream/ubuntu/jammy. Disregarding.

Reply:
I reordered this commit to sit on top of the other two I just added. Strange that github doesn't see that though.